src/utils.js

```diff
@@ -4,6 +4,10 @@ export const flatten = (array) => {
   return concat.apply([], array);
 };
 
+export const flatMap = (array, mapFn) => {
+  return array.reduce((nextArray, cur) => nextArray.concat(mapFn(cur)), []);
```

Comment:
Since there are instructions in the docs on how to use this library with browsers that don't natively have `Promise` or `Symbol` I felt I should avoid using `Array.prototype.flatMap` as none of those browsers would support it either.